src/main.cpp

```diff
@@ -1,16 +1,178 @@
 #include <iostream>
 #include <string>
 #include <BasisManager.h>
+#include <Base.h>
 #include <Potential.h>
-#include <Schroedinger.h>
+#include <Numerov.h>
+
+void box_potential_example() {
+	unsigned int nbox = 1000;
+	double mesh       = dx;
+	double k          = 0.0;
+	double height     = 0.0;
+	double width      = 0.0;
+	double energy     = 0.0;
+	std::vector<double> wavefunction;
+
+	// Initialize the base using preset and parameters
+	BasisManager::Builder baseBuilder;
+	Base base = baseBuilder.build(Base::basePreset::Cartesian, 1, mesh, nbox);
+	
+	Potential::Builder potentialBuilder(base);
+    Potential V = potentialBuilder.setType(Potential::PotentialType::BOX_POTENTIAL)
+								  .setK(k)
+								  .setHeight(height)
+								  .setWidth(width)
+								  .build();
```

Comment:
remove `setK, setHeight, setWidth` (all of them are 0 now). Add comment
`// no need to setup Box_potential, is just 0 and the above defined cartesian basis sets up the box. With zero-edge boundary conditions this correspond to infinite well potential with the width of the cartesian basis`

Reply:
It makes sense. Done!